### PR TITLE
Update to Djsv13 and improvements

### PR DESCRIPTION
**Pull request by danny519**
Mostly same behaviour but cleaner code and using djsv13 and slash commands.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,100 +1,100 @@
...
-
-			if (longerMatches.length > 0) embed.addField('Longer matches', longerMatches.map(match => `- \`${match.formatted}\``).join('\n'));
-			if (shorterMatches.length > 0) embed.addField('Shorter matches', shorterMatches.map(match => `- \`${match.formatted}\``).join('\n'));
-		}
-
-		message.channel.send('', embed);
-	}
+// const methodUsageEmbed = new Discord.MessageEmbed()
+//   .setTitle('Usage')
+//   .setDescription('```css\n.method <method_name> [usage_id]\n```');
```
Comment (Rami-Sabbagh) [inline src/index.ts]: Unnecessary commented code

File: package.json [hunk 2]
```diff
@@ -4,10 +4,15 @@
   "scripts": {
     "dev": "ts-node src/index.ts",
     "build": "tsc",
-    "start": "node dist/index.js"
+    "start": "node dist/index.js",
+    "devDeployCommands": "ts-node src/deploy-commands.js",
+    "deployCommands": "node dist/deploy-commands.js"
```
Comment (Rami-Sabbagh) [inline package.json]: I think the kebab-case convention is more nice than camelCase? it's totally fine, will solve that myself

**Review by Rami-Sabbagh — CHANGES_REQUESTED** [review]
Really well done, thanks for your effort Danny!!!

**Review by Rami-Sabbagh — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (danny519) [reply package.json]: I don't think it matters what we use as long as it's consistent.

File: src/deploy-commands.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import { SlashCommandBuilder, SlashCommandOptionsOnlyBuilder } from '@discordjs/builders';
```
Comment (danny519) [inline src/deploy-commands.ts:1]: I am not sure why I imported `SlashCommandOptionsOnlyBuilder` it does not look it's used anywhere.

Comment (Rami-Sabbagh) [reply src/deploy-commands.ts:1]: This makes me think of introducing `eslint` 😂

**Pull request merged**